Feature/id 517/kafka to mqtt transformer (#13)
hivemq/hivemq-kafka-extension-customization-sdk: README.adoc

The HiveMQ Kafka Extension SDK is licensed under the `APACHE LICENSE, VERSION 2.0`.
A copy of the license can be found link:LICENSE[here].

== Setup

> Make sure the `hivemq extension sdk` repository is checked out in the same folder as the customization sdk, so the hivemq

Reviewer: ```suggestion
Make sure the `hivemq-extension-sdk` repository is checked out in the same folder as the customization sdk, so the hivemq
```

> Make sure the `hivemq extension sdk` repository is checked out in the same folder as the customization sdk, so the hivemq
> extension sdk can be linked with this repository (see `settings.gradle.kts` file line `includeBuild("../hivemq-extension-sdk")`).

Reviewer: ```suggestion
To be able to use the latest change of the `hivemq-extension-sdk` please check out the https://github.com/hivemq/hivemq-extension-sdk repository in the same folder as the current project.
(See line `includeBuild("../hivemq-extension-sdk")` in `settings.gradle.kts`).
```

> NOTE: You can also change the path in the `settings.gradle.kts` to the path of your `hivemq extension sdk`.

Reviewer: ```suggestion
NOTE: You can also change the path in the `settings.gradle.kts` to the path of your `hivemq-extension-sdk`.
```
